R/bcf.R

```diff
@@ -110,7 +114,7 @@ Rcpp::loadModule(module = "TreeSamples", TRUE)
 #' @param sd_moderate SD(tau(x)) marginally at any covariate value (or its prior median if use_tauscale=TRUE)
 #' @param base_moderate Base for tree prior on tau(x) trees (see details)
 #' @param power_moderate Power for the tree prior on tau(x) trees (see details)
-#' @param save_tree_directory Specify where trees should be saved. Keep track of this for predict(). Defaults to working directory.
+#' @param save_tree_directory Specify where trees should be saved. Keep track of this for predict(). Defaults to working directory. Setting to NULL skips writing of trees.
 #' @param nu Degrees of freedom in the chisq prior on \eqn{sigma^2}
```

Comment:
@e-lipman  this looks great, and it tests great on my machine too! 

If you update the documentation or the default arguement thought, you need to run devtools::document() to have it update the appropiate markdown file!